pkgs/applications/graphics/inkscape/extensions.nix

```diff
@@ -8,6 +8,38 @@
 {
   applytransforms = callPackage ./extensions/applytransforms { };
 
+  living_hinge = stdenv.mkDerivation {
+    pname = "living_hinge";
+    version = "unstable-2020-01-21";
+
+    src = fetchFromGitHub {
+      owner = "stogo";
+      repo = "Inkscape_LivingHinge";
+      rev = "67df66c14130dfc831a3ce5c53d8d3b59427858b";
+      sha256 = "sha256-rmq3QUqS0iZRWPqe5j/nsDOCE1ZokzT76ZWIVq1YIH0=";
+    };
+
+    preferLocalBuild = true;
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p "$out/share/inkscape/extensions"
+      cp -p *.inx *.py "$out/share/inkscape/extensions/"
+      find "$out/share/inkscape/extensions/" -name "*.py" -exec chmod +x {} \;
+
+      runHook postInstall
+    '';
+
+    meta = with lib; {
```

Comment:
Maybe good to mark it 'broken' since it doesn't work with the inkscape in this nixpkgs, and document how to run it with an older inkscape?

It would be nice to track inkscape 1.2 support in an upstream issue, perhaps at https://github.com/drphonon/Inkscape_LivingHinge ?